chrome/content/zotero/xpcom/data/collection.js

```diff
@@ -23,7 +23,7 @@
     ***** END LICENSE BLOCK *****
 */
 
-Zotero.Collection = function() {
+Zotero.Collection = function(name) {
```

Comment:
I see in support.js we pass down `params` object as argument. Is that what we want to do in general?